src/components/TopBar/ProjectPicker.jsx

```diff
@@ -33,12 +46,37 @@ const ProjectPicker = createMenu({
         <ProjectPreview projectKey={projectKey} />
       </MenuItem>
     ));
+
+    if (!some(projects, 'isArchived')) {
```

Comment:
I think this is backwards—we want to show the archived view toggle if some of the projects _are_ archived.